ruleset/rules/0750-github_rules.xml

```diff
@@ -0,0 +1,3225 @@
+        <description>GitHub Dependency graph new repos enable.</description>
+        <options>no_full_log</options>
+        <group>git_dependency_graph_new_repos</group>
+    </rule>
+
+    <!-- discussion_post
+            update
+            destroy
+    -->
+    <rule id="91132" level="3">
+        <if_sid>91100</if_sid>
+        <action type="osregex">^discussion_post.\.*</action>
+        <description>GitHub Discussion post.</description>
+        <options>no_full_log</options>
+        <group>git_discussion_post</group>
+    </rule>
+
+    <!-- discussion_post.update -->
+    <rule id="91133" level="5">
+        <if_sid>91132</if_sid>
+        <action>discussion_post.update</action>
+        <description>GitHub Discussion post update.</description>
+        <options>no_full_log</options>
+        <group>git_discussion_post</group>
+    </rule>
+
+    <!-- discussion_post.destroy -->
+    <rule id="91134" level="5">
+        <if_sid>91132</if_sid>
+        <action>discussion_post.destroy</action>
+        <description>GitHub Discussion post destroy.</description>
+        <options>no_full_log</options>
+        <group>git_discussion_post</group>
+    </rule>
+
+    <!-- discussion_post_reply
+            update
+            destroy
+    -->
+    <rule id="91135" level="3">
+        <if_sid>91100</if_sid>
+        <action type="osregex">^discussion_post_reply.\.*</action>
+        <description>GitHub Discussion post reply.</description>
+        <options>no_full_log</options>
+        <group>git_discussion_post_reply</group>
+    </rule>
+
+    <!-- discussion_post_reply.update -->
+    <rule id="91136" level="5">
+        <if_sid>91135</if_sid>
+        <action>discussion_post_reply.update</action>
+        <description>GitHub Discussion post replay update.</description>
+        <options>no_full_log</options>
+        <group>git_discussion_post_reply</group>
+    </rule>
+
+    <!-- discussion_post_reply.destroy -->
+    <rule id="91137" level="5">
+        <if_sid>91135</if_sid>
+        <action>discussion_post_reply.destroy</action>
+        <description>GitHub Discussion post replay destroy.</description>
+        <options>no_full_log</options>
+        <group>git_discussion_post_reply</group>
+    </rule>
+
+    <!-- enterprise
+            remove_self_hosted_runner
+            register_self_hosted_runner
+            runner_group_created
+            runner_group_removed
+            runner_group_runner_removed
+            runner_group_runners_added
+            runner_group_runners_updated
+            runner_group_updated
+            self_hosted_runner_updated
+    -->
+    <rule id="91139" level="3">
+        <if_sid>91100</if_sid>
+        <action type="osregex">^enterprise.\.*</action>
+        <description>GitHub Enterprise.</description>
+        <options>no_full_log</options>
+        <group>git_enterprise</group>
+    </rule>
+
+    <!-- enterprise.remove_self_hosted_runner -->
+    <rule id="91140" level="7">
```

Comment:
level 7 should be used for “Bad word” matching, not sure what does removing self hosted runner implies